fmt/format.h

```diff
@@ -2405,6 +2422,7 @@ class SystemError : public internal::RuntimeError {
   SystemError(int error_code, CStringRef message) {
     init(error_code, message, ArgList());
   }
+  FMT_DEFAULTED_COPY_CTOR(SystemError);
```

Comment:
I suggest moving semicolon to the definition `FMT_DEFAULTED_COPY_CTOR` to prevent it dangling if the latter is defined to empty.